WordPress/Classes/ViewRelated/Blog/Site Management/SiteSettingsViewController+SiteManagement.swift

```diff
@@ -46,27 +46,31 @@ public extension SiteSettingsViewController {
         SVProgressHUD.show(withStatus: status)
 
         let trackedBlog = blog
-        WPAppAnalytics.track(.siteSettingsExportSiteRequested, with: trackedBlog)
+        if let trackedBlog {
+            WPAppAnalytics.track(.siteSettingsExportSiteRequested, blog: trackedBlog)
+        }
```

Comment:
I think we can add an `else` block which runs `WPAppAnalytics.track(.siteSettingsExportSiteRequested)` here? Same as the `if let trackedBlog` statements below?

Reply:
Good point. I made `blog` optional in `WPAppAnalytics` for convenience and updated all of these method invocations.